pipelines/batch/src/main/java/org/openmrs/analytics/FhirSearchUtil.java

```diff
@@ -61,6 +70,39 @@ public Bundle searchByUrl(String searchUrl, int count, SummaryEnum summaryMode)
 		return null;
 	}
 	
+	/**
+	 * Searches for the total number of resources for each resource type and return a HashMap
+	 */
+	public HashMap<String, Integer> searchResourceCounts(String searchUrl, HashSet<String> resourceTypes) {
+		HashMap<String, Integer> hashMap = new HashMap<String, Integer>();
+		HttpResponse response;
+		DefaultHttpClient httpClient = new DefaultHttpClient();
+		HttpGet getConnection = new HttpGet(searchUrl);
+		
+		try {
+			response = httpClient.execute(getConnection);
```

Comment:
When possible, please try to use FHIR client similar to the function above (instead of creating FHIR URLs directly).